internal/providers/terraform/azure/testdata/hdinsight_hadoop_cluster_test/hdinsight_hadoop_cluster_test.tf

```diff
@@ -35,7 +35,6 @@ resource "azurerm_hdinsight_hadoop_cluster" "with_edge" {
   }
 
   gateway {
-    enabled  = true
```

Comment:
This and other changes: deprecated and removed https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/guides/3.0-upgrade-guide#resource-azurerm_hdinsight_hadoop_cluster